packages/webdriverio/src/scripts/resq.js

```diff
@@ -3,35 +3,25 @@ export const waitToLoadReact = function waitToLoadReact () {
 }
 
 export const react$ = function react$ (selector, props, state) {
-    let elems = window.resq.resq$$(selector)
+    let elem = window.resq.resq$(selector)
```

Comment:
It might be clearer to use full word `element` instead of `elem`.

Reply:
I agree. Changed d9cc9c6